pybamm/models/full_battery_models/lithium_ion/basic_dfn_half_cell.py

```diff
@@ -123,7 +127,7 @@ def __init__(self, options=None, name="Doyle-Fuller-Newman half cell model"):
                 pybamm.Parameter("Separator porosity"), "separator"
             )
             eps_w = pybamm.PrimaryBroadcast(
-                pybamm.Parameter("Negative electrode porosity"), "working electrode"
+                pybamm.Parameter("Negative electrode porosity"), "positive electrode"
```

Comment:
After reviewing the PR I have realised that my previous comment only applies to "half cell" which is the only scenario where we might want to flip the electrodes so when that functionality is implemented it can take care of that (and probably then we can get rid of `BasicDFNHalfCell`)